assets/src/dashboard/templates/README.md

```diff
@@ -0,0 +1,26 @@
+# Template Generation From Raw Story JSON
+
+`templates/raw` is a dumping ground for raw story json output from the editor, which is then interpreted as a template.
+
+## Getting The Raw Story JSON
+
+To get the raw story json, simply create a new story in the web story editor. Open your network tab and save the story. You should see a network request that contains a large json, this json should have a field in it named `story_data`.
```

Comment:
You might not be aware of the DevTools we've recently added to the editor:

https://github.com/google/web-stories-wp/blob/8e7d3fd1f1113d4d41544b1261081e9390d43c94/docs/devtools.md

In the editor, you can press `Command+Shift+Option+J / Control+Shift+Alt+J` and then the story data will be displayed in a textarea. This should be much easier for creating templates.

cc @merapi to confirm

----

On that note, might be useful to move this doc to `/docs`?

Reply:
@swissspidy , @mariano-formidable just pointed out to me that this story data text area is missing some fields that are typically found in `story_data` like version number.

Isn't this something we're going to need when auto converting template versions?

Reply:
Indeed it is.

cc @merapi any chance that could be added? or any reason why not?

Reply:
@littlemilkstudio 
`story_data` field structure and what we currently have in DevTools are slightly different (extra fields/missing fields), so the question is: Will just adding the version work for your use case? (easy change) or we need to create a checkbox that will show data in the exact `story_data` field format? (bigger change)